fault-proof/docker-compose.yml

```diff
@@ -13,10 +13,56 @@ services:
   challenger:
     build:
       context: ..
-      dockerfile: fault_proof/Dockerfile.challenger
+      dockerfile: fault-proof/Dockerfile.challenger
     image: op-succinct-fp-challenger:latest
     container_name: op-succinct-fp-challenger
     restart: unless-stopped
     env_file:
       - .env.challenger
     command: ["challenger"]
+
+  prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus
+    restart: unless-stopped
+    ports:
+      - "${FP_PROMETHEUS_PORT:-9090}:9090"
+    volumes:
+      - ./prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
+      - /home/op_succinct_lite_prometheus_data:/prometheus
+    user: root
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+      - '--web.console.libraries=/etc/prometheus/console_libraries'
+      - '--web.console.templates=/etc/prometheus/consoles'
+      - '--web.enable-lifecycle'
+
+  grafana:
+    image: grafana/grafana:latest
+    container_name: grafana
+    restart: unless-stopped
+    ports:
+      - "${FP_GRAFANA_PORT:-3000}:3000"
+    volumes:
+      - ./grafana/provisioning:/etc/grafana/provisioning
+      - ./grafana/dashboards:/var/lib/grafana/dashboards
+      - ./grafana/alerts:/var/lib/grafana/alerts
+      - /home/op_succinct_lite_grafana_data:/var/lib/grafana
+    user: root
+    environment:
+      - GF_SECURITY_ADMIN_USER=admin
+      - GF_SECURITY_ADMIN_PASSWORD=admin
+      - GF_USERS_ALLOW_SIGN_UP=false
+
+volumes:
+  prometheus_data:    
+    driver_opts:
+      type: none
+      device: /home/op_succinct_lite_prometheus_data
+      o: bind
+  grafana_data:
+    driver_opts:
+      type: none
+      device: /home/op_succinct_lite_grafana_data
+      o: bind
```

Comment:
Let's not hardcode the volumes to `/home` here and instead use the defaults. In our config we can set `/home`.